plugins/mtd/meson.build

```diff
@@ -0,0 +1,29 @@
+cargs = ['-DG_LOG_DOMAIN="FuPluginMtd"']
```

Comment:
I think we don't want to bother building this for other OSes besides Linux - shouldn't it have a guard?